src/ui/List.tsx

```diff
@@ -72,6 +72,11 @@ const ListItemContent = styled.div<SpacedList>`
       padding-left: 1.5rem;
     }
   }
+  div + svg {
```

Comment:
@mxcosu and I were chatting about this CSS this morning and thought it'd be nice to have a comment about why this is here (targetting when the div has an adjacent sibling svg and that it is to make the warning icon location and sizing specific)